### PR TITLE
Use native `--version` support in `urfave/cli/v2`

### LINKED ISSUE #96: `scip --version` reports 0.1.0
There are some problems with `scip` command line arguments. I've installed v0.2.1.

```sh
$ ./scip --version
0.1.0
$ ./scip version
$ ./scip verwhat
$
```

### PR DESCRIPTION
**Pull request by abitrolly**
https://cli.urfave.org/v2/examples/version-flag/

Fixes #96 (until next time).

### CODE REVIEW

**Comment by varungandhi-src** [comment]
Added some notes on what I think an ideal solution to keeping these in sync would look like: https://github.com/sourcegraph/scip/issues/99

**Comment by varungandhi-src** [comment]
Sorry, Prettier is being pedantic. 😮‍💨 If it's any consolation, I get that stupid error too a bunch.

**Comment by abitrolly** [comment]
I can't fix that. Prettier complains about trailing spaces and test compares them.

**Comment by varungandhi-src** [comment]
`npm run prettier` should fix it

**Comment by abitrolly** [comment]
@varungandhi-src `npm run prettier` will remove the trailing whitespace from `Readme.md`, but tests compare this to `--help` output, which contains the whitespace. I fixed tests until `urfave/cli` fix is merged (PR above).

**Comment by varungandhi-src** [comment]
Hmm, the whitespace issue is coming up now but hasn't come up before -- is it because of the urfave/cli bump?

**Comment by abitrolly** [comment]
> Hmm, the whitespace issue is coming up now but hasn't come up before -- is it because of the urfave/cli bump?

Yes, added in v2.10.0 - https://github.com/urfave/cli/commit/4fd56cb22d4640b243dbb9a77c0b92d72a92ac87 and https://github.com/urfave/cli/releases/tag/v2.10.0

File: cmd/main_test.go [hunk 1]
```diff
@@ -39,6 +40,8 @@ func TestReadmeInSync(t *testing.T) {
 		helpBytes = helpBytes[:n-1] // ignore NULL at end
 		require.Nil(t, err)
 		help := strings.TrimSpace(string(helpBytes))
+		re := regexp.MustCompile(`(?m)\s+$`) // strip trailing whitespace
+		help = re.ReplaceAllString(help, "\n")
```
Comment (abitrolly) [inline cmd/main_test.go:44]: These lines can be removed when new `urfave/cli` 2.17.2+ is out.

Comment (abitrolly) [reply cmd/main_test.go:44]: https://github.com/sourcegraph/scip/pull/110

**Pull request merged**